djangorestframework_camel_case/util.py

```diff
@@ -37,10 +37,10 @@ def camelize(data, **options):
                 new_key = re.sub(camelize_re, underscore_to_camel, key)
             else:
                 new_key = key
-            if key not in ignore_fields and new_key not in ignore_fields:
```

Comment:
I think this is there for a reason, it's so you can add either the (prospective) new key, or the old key, to a list of keys to ignore.

Reply:
hmm, if that's the case I would still rewrite it as:

```
if key in ignore_fields or new_key in ignore_fields:
     new_dict[key] = value
else:
    new_dict[new_key] = camelize(value, **options)
```

It's a bit easier to read IMO